transform/voxel_around_line.go

```diff
@@ -33,7 +33,7 @@ import (
 //	拡張空間IDスライス： []string
 //  error: エラー
 
-func GetSpatialIdsWithinRadiusOfLine(startPoint *object.Point, endPoint *object.Point, radius float64, hZoom int64, vZoom int64, skipsMeasurement bool) ([]string, error) {
+func GetExtendedSpatialIdsWithinRadiusOfLine(startPoint *object.Point, endPoint *object.Point, radius float64, hZoom int64, vZoom int64, skipsMeasurement bool) ([]string, error) {
```

Comment:
This change is incompatible.
Change over the module name from v2 to v3.

Reply:
resolved here https://github.com/trajectoryjp/spatial_id_go/pull/19/commits/faaea39b58feb9951abf3ab1d1e1bb0df056b5c6